packages/aws-cdk/CONTRIBUTING.md

```diff
@@ -19,6 +19,17 @@ Run:
 npm run integ-cli
 ```
 
+In addition, you can run CLI integration compatibility tests. 
```

Comment:
I would actually expect `npm run integ-cli` (which should be changed to `yarn integ-cli`) to include those tests. We can introduce `yarn integ-cli-no-regression` if someone insists on running the "old" set. But running all the integ tests of the CLI should be `yarn integ-cli` in my mind.

Elaborate what is the rational behind each type of regression test.

Reply:
👍